packages/assets/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "@atb-as/generate-assets",
+  "version": "0.0.1",
+  "private": false,
+  "description": "OOS Design System generate-assets",
+  "license": "EUPL-1.2",
+  "bin": "./lib/bin.js",
+  "main": "./lib/generate.js",
+  "files": [
+    "lib"
+  ],
+  "scripts": {
+    "prepublish": "tsc"
+  },
+  "devDependencies": {},
```

Comment:
Jeg tror her egentlig vi burde ha `@types/node` og `typescript` som avhengigheter til den lokale pakken. Særlig med tanke på at vi bruker `tsc` som prepublish-steg. Typescript er en avhengighet til pakken, så den burde deklareres som en `devDependency`.

Eksempel for å legge til (fra repo root):

```
yarn workspace @atb-as/generate-assets add -D @types/node typescript
```